### ISSUE TITLE
Input goes into different window when changing languages

### ISSUE THREAD

**Issue by CookieHCl** [microsoft/terminal#3972]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18363.535]
Windows Terminal version (if applicable): Version: 0.7.3451.0
```
Windows Terminal was installed by chocolatey.

# Steps to reproduce

1. Open Windows Terminal.
2. Use 한영키(The key that switches English <-> Korean) to change input to Korean.
3. Open new window.
4. Type any Korean. (Let's say we input `가나다`)

# Expected behavior

Of course, Korean characters must be inputed into new window.

# Actual behavior

Sometimes Korean characters are inputed into previous window.
(If you open multiple windows at step 3, input goes into one of the window you open but I'm not sure which one is it.)

This only happens with Korean characters. More precisely, in this bug state all Korean characters will be inputed into previous window and all non-Korean characters such as numbers `123` or symbols `!@#` will be inputed into current window.

# Sometimes?

Yes. Unfortunately, this bug sometimes appears and I couldn't figure out the exact condition. But it happens quite often, and if you failed reproducing bug, you can just try again by opening new window. (You don't have to turn off whole windows terminal, but turning off is better)

This gif shows the bug when you open one window and when you open multiple window.

![ezgif-2-a3982cece4bc](https://user-images.githubusercontent.com/28638469/70863306-1797e580-1f8a-11ea-9990-fd9ec2ddf178.gif)

**Comment by leonMSFT** [comment]
@CookieHCl We've recently been pushing a bunch of IME changes, and I can't seem to repro the issue you've shown above. I'm wondering if you can check if you can still repro this on the latest build?

**Comment by CookieHCl** [comment]
@leonMSFT I've checked again and it seems this issue is fixed in v0.9.433.0(or v0.8.10261.0. This issue was reproducible at v0.8.10091.0). You can close this issue.

**Comment by leonMSFT** [comment]
Great, thanks for the follow up! 😄
